### PR TITLE
BXMSDOC-2170-master: Add sections for system properties

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 1]
```diff
@@ -43,26 +81,27 @@ NOTE: If the default or assigned port is already in use, a new port is automatic
...
-* `org.uberfire.secure.key`: Password used by password encryption. Default value: `org.uberfire.admin`.
-* `org.uberfire.secure.alg`: Crypto algorithm used by password encryption. Default value: `PBEWithMD5AndDES`.
-* `org.uberfire.domain`: Security-domain name used by uberfire. Default value: `ApplicationRealm`.
-* `org.guvnor.m2repo.dir`: Place where the Maven repository folder is stored. Default value: `<working-directory>/repositories/kie`.
-* `org.guvnor.project.gav.check.disabled`: Disables group ID, artifact ID, and version (GAV) checks. Default value: `false`.
-* `org.kie.build.disable-project-explorer`: Disables automatic build of a selected project in Project Explorer. Default value: `false`.
-* `org.kie.verification.disable-dtable-realtime-verification`: Disables the real-time validation and verification of decision tables. Default value: `false`.
+{KIE_SERVER} nodes and {CONTROLLER}::
+
+Use these properties to configure how to connect to the {KIE_SERVER} nodes from the {CONTROLLER}.
```
Comment (hmanwani-rh) [inline doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:95]: Suggested rewrite:
Use the following properties to configure the connections with the {KIE_SERVER} nodes from the {CONTROLLER}:

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 2]
```diff
@@ -11,22 +11,56 @@ java -jar {PRODUCT_FILE}-{URL_COMPONENT_CENTRAL}-standalone.jar -s application-c
...
 * `org.uberfire.nio.git.http.enabled`: Enables or disables the HTTP daemon. Default value: `true`.
 * `org.uberfire.nio.git.http.host`: If the HTTP daemon is enabled, it uses this property as the host identifier. This is an informative property that is used to display how to access the Git repository over HTTP.  The HTTP still relies on the servlet container. Default value: `localhost`.
 * `org.uberfire.nio.git.http.hostname`: If the HTTP daemon is enabled, it uses this property as the host name identifier. This is an informative property that is used to display how to access the Git repository over HTTP.  The HTTP still relies on the servlet container. Default value: `localhost`.
 * `org.uberfire.nio.git.http.port`: If the HTTP daemon is enabled, it uses this property as the port number. This is an informative property that is used to display how to access the Git repository over HTTP.  The HTTP still relies on the servlet container. Default value: `8080`.
+
+Git over HTTPS::
+
+Use these properties to set how to access the Git repository over HTTPS.
+
+* `org.uberfire.nio.git.proxy.ssh.over.https`: Specifies whether SSH should use an HTTPS proxy. Default value: `false`.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:41]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 3]
```diff
@@ -43,26 +81,27 @@ NOTE: If the default or assigned port is already in use, a new port is automatic
 NOTE: If the default or assigned port is already in use, a new port is automatically selected. Ensure that the ports are available and check the log for more information.
 
 * `org.uberfire.nio.git.ssh.cert.dir`: Location of the `.security` directory where local certificates are stored. Default value: Working directory.
+* `org.uberfire.nio.git.ssh.idle.timeout`: Sets the ssh idle timeout.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:84]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 4]
```diff
@@ -74,6 +113,27 @@ NOTE: Use this property for {CENTRAL} only. If you share a runtime environment w
 * `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the {CENTRAL} `.security` folder.
 * `appformer.experimental.features`: Enables the experimental features framework. Default value: `false`.
 * `org.kie.demo`: Enables an external clone of a demo application from GitHub.
+
+
+
+
+
+* `org.uberfire.metadata.index.dir`: Place where the Lucene `.index` directory is stored. Default value: Working directory.
+* `org.uberfire.ldap.regex.role_mapper`: Regex pattern used to map LDAP principal names to the application role name. Note that the variable role must be part of the pattern because it is substituted by the application role name when matching a principal value to a role name.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:122]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 5]
```diff
@@ -11,22 +11,56 @@ java -jar {PRODUCT_FILE}-{URL_COMPONENT_CENTRAL}-standalone.jar -s application-c
...
+* `https.proxyPassword`: Defines the user password of the HTTPS proxy.
+* `user.dir`: Location of the user directory.
 * `org.uberfire.nio.git.https.enabled`: Enables or disables the HTTPS daemon. Default value: `false`
 * `org.uberfire.nio.git.https.host`: If the HTTPS daemon is enabled, it uses this property as the host identifier. This is an informative property that is used to display how to access the Git repository over HTTPS.  The HTTPS still relies on the servlet container. Default value: `localhost`.
 * `org.uberfire.nio.git.https.hostname`: If the HTTPS daemon is enabled, it uses this property as the host name identifier. This is an informative property that is used to display how to access the Git repository over HTTPS.  The HTTPS still relies on the servlet container. Default value: `localhost`.
 * `org.uberfire.nio.git.https.port`: If the HTTPS daemon is enabled, it uses this property as the port number. This is an informative property that is used to display how to access the Git repository over HTTPS.  The HTTPS still relies on the servlet container. Default value: `8080`.
+
+JGit::
+
+* `org.uberfire.nio.jgit.cache.instances`:
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:54]: added: "Defines the JGit cache size."

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 6]
```diff
@@ -11,22 +11,56 @@ java -jar {PRODUCT_FILE}-{URL_COMPONENT_CENTRAL}-standalone.jar -s application-c
 In this command, `<property>` is a property from list and `<value>` is a value that you assign to that property.
 endif::[]
 
-* `org.uberfire.nio.git.dir`: Location of the {KIE_SERVER} Git directory.
-* `org.uberfire.nio.git.dirname`: Name of the {KIE_SERVER} Git directory. Default value: `.niogit`.
+Git directory::
+
+Use these properties to set the location and name for the {CENTRAL} Git directory.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:16]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc:95]: Done

File: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc [hunk 7]
```diff
@@ -74,6 +113,27 @@ NOTE: Use this property for {CENTRAL} only. If you share a runtime environment w
...
+* `org.uberfire.domain`: Security-domain name used by uberfire. Default value: `ApplicationRealm`.
+* `org.guvnor.m2repo.dir`: Place where the Maven repository folder is stored. Default value: `<working-directory>/repositories/kie`.
+* `org.guvnor.project.gav.check.disabled`: Disables group ID, artifact ID, and version (GAV) checks. Default value: `false`.
+* `org.kie.build.disable-project-explorer`: Disables automatic build of a selected project in Project Explorer. Default value: `false`.
+* `org.kie.verification.disable-dtable-realtime-verification`: Disables the real-time validation and verification of decision tables. Default value: `false`.
+
+
+{CONTROLLER}::
+
+Use these properties to configure how to connect to the {CONTROLLER}.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc]: How about? "Use the following properties to configure the connection to the {CONTROLLER}:"

**Comment by csherrar** [comment]
> @csherrar Have added a few comments below. Rest looks good
> Also, some of the sentences as descriptions are incomplete, and should not be ending with a period. therefore, as we agreed it would be great if the sentences can be completed.
> Let me know if you have any questions.

Thank you!

**Pull request merged**